apps/configurator/README.md

```diff
@@ -35,6 +35,37 @@ You can:
 - Assign skills (see skills config) to each character: basic, ultimate, dash.
 - Tune character params: speed, health, stamina, etc.
 
+Its mandatory to activate a character to have a basic skin assigned, for example, if you want to add Uren you should add this this lines of code:
+
+- Set `active` to `true` in `seeds.exs`
+charactername_params = %{
+  name: "valtimer",
+  active: true,
+  ...
+  ...
+}
```

Comment:
A few things here:
- This README is for the Configurator app. It's documentation to add/edit characters via the forms in `localhost:4100`, not the seeds.
- You can either add a character via Configurator or by code using seeds.
The `active` param is the one that enables and disables characters, but it's already been said above:
> - Change it's `active` value to enable/disable characters in game.
- Also, you talk about Uren but the params are for Valtimer.